src/EnergyPlus/Coils/CoilCoolingDX.cc

```diff
@@ -980,99 +980,162 @@ void CoilCoolingDX::passThroughNodeData(DataLoopNode::NodeData &in, DataLoopNode
+                OutputReportPredefined::PreDefTableEntry(
+                    state, state.dataOutRptPredefined->pdchDXCoolCoilSEERUserIP, coil.name, coil.performance.standardRatingSEER * ConvFromSIToIP, 2);
+            } else {
+                OutputReportPredefined::PreDefTableEntry(state, state.dataOutRptPredefined->pdchDXCoolCoilSEERUserIP, coil.name, "N/A");
+            }
+            if (coil.performance.standardRatingSEER_Standard > 0.0) {
+                OutputReportPredefined::PreDefTableEntry(state,
+                                                         state.dataOutRptPredefined->pdchDXCoolCoilSEERStandardIP,
+                                                         coil.name,
+                                                         coil.performance.standardRatingSEER_Standard * ConvFromSIToIP,
+                                                         2);
+            } else {
+                OutputReportPredefined::PreDefTableEntry(state, state.dataOutRptPredefined->pdchDXCoolCoilSEERStandardIP, coil.name, "N/A");
+            }
+            if (coil.performance.standardRatingIEER > 0.0) {
+                OutputReportPredefined::PreDefTableEntry(
+                    state, state.dataOutRptPredefined->pdchDXCoolCoilIEERIP, coil.name, coil.performance.standardRatingIEER * ConvFromSIToIP, 1);
+            } else {
+                OutputReportPredefined::PreDefTableEntry(state, state.dataOutRptPredefined->pdchDXCoolCoilIEERIP, coil.name, "N/A");
+            }
             OutputReportPredefined::addFootNoteSubTable(
                 state,
                 state.dataOutRptPredefined->pdstDXCoolCoil,
                 "ANSI/AHRI ratings account for supply air fan heat and electric power. <br/>"
                 "1 - EnergyPlus object type. <br/>"
-                "2 - Capacity less than 65K Btu/h - calculated as per AHRI Standard 210/240-2017. <br/>"
-                "&emsp;&nbsp;Capacity of 65K Btu/h to less than 135K Btu/h - calculated as per AHRI Standard 340/360-2007. <br/>"
-                "&emsp;&nbsp;Capacity 135K Btu/h or more - n/a - should be calculated as per AHRI standard 365-2009. <br/>"
+                "2 - Capacity less than 65K Btu/h (19050 W) - calculated as per AHRI Standard 210/240-2017. <br/>"
+                "&emsp;&nbsp;Capacity of 65K Btu/h (19050 W) to less than 135K Btu/h (39565 W) - calculated as per AHRI Standard 340/360-2007. "
+                "<br/>"
+                "&emsp;&nbsp;Capacity from 135K (39565 W) to 250K Btu/hr (73268 W) - calculated as per AHRI Standard 365-2009 - Ratings not yet "
+                "supported in EnergyPlus. <br/>"
                 "3 - SEER (User) is calculated using user-input PLF curve and cooling coefficient of degradation. <br/>"
                 "&emsp;&nbsp;SEER (Standard) is calculated using the default PLF curve and cooling coefficient of degradation"
                 "from the appropriate AHRI standard.");
```

Comment:
So, it looks like there are 4 places in the code all writing the same set of footnotes? One here, one in DXCoils.cc, and two in StandardRatings.cc. Should only need this code on one place.

Reply:
Yes, this could be refactored.  We are investigating, but could this be included in a new SOW that addresses all of the issues you have raised?